.github/CODEOWNERS

```diff
@@ -39,8 +43,8 @@ README.md @iamrecursion @kustosz
 
 # Config
 /.github @iamrecursion @kustosz
-/ci @iamrecursion @kustosz
```

Comment:
`ci` still exists, but it's now in `tools/`. You can make me, marcin, and yourself reviewers for the whole `tools/` directory.